docs/features/software-catalog/descriptor-format.md

```diff
@@ -312,6 +316,14 @@ Apart from being a string, the software catalog leaves the format of this field
 open to implementers to choose. Most commonly, it is set to the ID or email of a
 group of people in an organizational structure.
 
+### `spec.implementsApis` [optional]
```

Comment:
Yes. Since [ADR 005](https://github.com/spotify/backstage/blob/master/docs/architecture-decisions/adr005-catalog-core-entities.md#decision) also intends the "consumes" relation it might be good to have a more general approach in the future. As soon as some kind of versioning is introduced, this simple modeling will also not suffice. But we will see then.